openapi-generator/templates/java/libraries/okhttp-gson/api.mustache

```diff
@@ -100,7 +100,12 @@ public class {{classname}} {
         {{javaUtilPrefix}}List<Pair> localVarCollectionQueryParams = new {{javaUtilPrefix}}ArrayList<Pair>();
         {{#queryParams}}
         if ({{paramName}} != null) {
+            {{#isMap}}
```

Comment:
I believe you should use `#isDeepObject` instead. You can see it's available to the API template if you run generate with `--global-property debugOperations=true` (also `isExplode` is available)

Reply:
Thanks, will take a look, the document it printed out for me upon running with this argument,

- niether included any documentations for get end points
- nor `isDeepObject` or `isExplode` properties

But let me check again, would be more specific

Reply:
or, you can pass `{{style}}` as a parameter to `mappedParameterToPairs`, and then handle it appropriately (leaving other styles not covered until we need them).

Reply:
note that there are different `debug...` flags possible to pass as `global-property`. https://openapi-generator.tech/docs/debugging/#templates

Reply:
Will check this. Thanks for bringing it up.

Reply:
Thanks Mladen, using `debugOperations` was able to find the name of the directive.